### ISSUE TITLE
Suggestions Page

### ISSUE THREAD

**Issue by amcquade** [amcquade/quick-feed#8]
Leave a comment if you would like to add a feature or something like that

**Comment by SinghVikram97** [comment]
How to give suggestions? Any recommender system api available?

**Comment by amcquade** [comment]
@SinghVikram97 not yet, want to work on that ?

**Comment by SinghVikram97** [comment]
Yes but no idea about Machine Learning part I only have knowledge of WebDev

**Comment by hrishibawane** [comment]
@amcquade  hey, I would like to work on this project. Can you provide some insights or something like a to-do you have in mind?

**Comment by AnirudhSriram** [comment]
> 
> 
> Yes but no idea about Machine Learning part I only have knowledge of WebDev

I think for starters, you can help out by defining the inputs for a recommendation to be made such as
 * History of feeds searched 
 * Time spent on each podcast 
Building your own recommendation engine is in itself a mammoth job.

**Comment by amcquade** [comment]
@AnirudhSriram I agree. ML would be pretty out of scope for this simple project..this issue was originally created for a feature suggestion page

**Comment by MargotRasamy** [comment]
Hello @amcquade, with my friend @salamisodikiolawale, we would like to implement a feature to add the rss feeds to a list of favorites. It would be stored on local storage so his favorites are saved on his browser.
